draft-linus-trans-gossip-ct.md

```diff
@@ -149,7 +149,7 @@ There are three separate gossip streams:
 
 - Trusted Auditor Stream, HTTPS clients communicating directly with
   trusted CT auditors/monitors sharing SCTs, certificate chains and
-  STHs.
+  STHs. [Isn't this also done by HTTPS servers?]
```

Comment:
I think an HTTPS server having a trusted auditor relationship is outlined (although we didn't call it that explicitly): HTTPS servers can have a "push" mechanism to share SCTs they receive or a "pull" mechanism to let auditors request the SCTs they've received.